homeworks/hw2/stackCalculator/stackCalculator/StackOnLinkedList.cs

```diff
@@ -0,0 +1,71 @@
+﻿namespace StackCalculator;
+
+using System.Collections;
+using System.Collections.Generic;
+
+// Stack implementation using a linked list
+internal class StackOnLinkedList : IStack
+{
+
+    public StackOnLinkedList() 
```

Comment:
```suggestion
{
    public StackOnLinkedList() 
```
Хотя на самом деле тут должен быть комментарий :)